imagetest/cmd/manager/main.go

```diff
@@ -13,6 +13,7 @@ import (
 	"cloud.google.com/go/storage"
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/test_suites/disk"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/test_suites/fioperf"
```

Comment:
I think we should create a generic performance test suite and then within it have the different tests for storage, networking, etc. Also, let's use storage_perf instead of fio so it is inclusive of other tools (fio is just one possible tool).

Reply:
renamed to storage_perf